analytics/views.py

```diff
@@ -331,70 +334,50 @@ def user_activity_intervals():
 def sent_messages_report(realm):
     # type: (str) -> str
     title = 'Recently sent messages for ' + realm
+    realm_id = get_realm(realm)
 
     cols = [
         'Date',
         'Humans',
         'Bots'
     ]
 
-    query = '''
-        select
-            series.day::date,
-            humans.cnt,
-            bots.cnt
-        from (
-            select generate_series(
-                (now()::date - interval '2 week'),
-                now()::date,
-                interval '1 day'
-            ) as day
-        ) as series
-        left join (
-            select
-                pub_date::date pub_date,
-                count(*) cnt
-            from zerver_message m
-            join zerver_userprofile up on up.id = m.sender_id
-            join zerver_realm r on r.id = up.realm_id
-            where
-                r.domain = %s
-            and
-                (not up.is_bot)
-            and
-                pub_date > now() - interval '2 week'
-            group by
-                pub_date::date
-            order by
-                pub_date::date
-        ) humans on
-            series.day = humans.pub_date
-        left join (
-            select
-                pub_date::date pub_date,
-                count(*) cnt
-            from zerver_message m
-            join zerver_userprofile up on up.id = m.sender_id
-            join zerver_realm r on r.id = up.realm_id
-            where
-                r.domain = %s
-            and
-                up.is_bot
-            and
-                pub_date > now() - interval '2 week'
-            group by
-                pub_date::date
-            order by
-                pub_date::date
-        ) bots on
-            series.day = bots.pub_date
-    '''
-    cursor = connection.cursor()
-    cursor.execute(query, [realm, realm])
-    rows = cursor.fetchall()
-    cursor.close()
+    today = datetime.today().replace(tzinfo=timezone.utc) + timedelta(days=1)
```

Comment:
We should just process till timezone.now(), since once we process a countstat at a certain end_time, we never do so again (due to the FillState table).